src/components/DotNavigation/docs/DotNavigationExtendedExample.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import extendedStyles from './DotNavigationExtendedExample.st.css';
+import { DotNavigation, DotNavigationProps } from '../DotNavigation';
+
+export class DotNavigationExtendedExample extends React.Component<
```

Comment:
Let's add a `currentIndex` state prop, and change it with the `onSelect`, so that we can play with the example

Reply:
Done)